### DISCUSSION TITLE
Problem clonning repo on windows

### DISCUSSION THREAD

**Discussion by crisfcodes** [saleor/saleor#8940] [General]
I'm having problems cloning the saleor repo on windows, if you try to clone directly, the console trhown an error: `error: invalid path 'saleor/graphql/core/tests/cassettes/test_get_oembed_data[http:/www.youtube.com/watch?v=dQw4w9WgXcQ-VIDEO].yaml' fatal: unable to checkout working tree warning: Clone succeeded, but checkout failed.`

How to solve this? I've tried with `git config --global core.protectNTFS false` according some stackoverflow answers but doesn't work...

**Comment by slamer59** [comment]
Hello, I have made a [PR ](https://github.com/saleor/saleor/pull/9086 )to start solving this. 
I take one option to just rename files and related folder but I am not sure it is used. 

It seems to be tested in another way here:
https://github.com/saleor/saleor/blob/237a8cce5fdc88d2995903c4c872344669f6fd08/saleor/graphql/core/tests/test_core.py
